CONTRIBUTING.md

```diff
@@ -1,31 +1,32 @@
-# Contributing to Project Helix
 
-This project (like almost all of Project Helix) is an Open Development project and welcomes contributions from everyone who finds it useful or lacking.
 
-## Code Of Conduct
-
-This project adheres to the Adobe [code of conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to cstaub at adobe dot com.
-
-## Contributor License Agreement
-
-All third-party contributions to this project must be accompanied by a signed contributor license. This gives Adobe permission to redistribute your contributions as part of the project. [Sign our CLA](http://opensource.adobe.com/cla.html)! You only need to submit an Adobe CLA one time, so if you have submitted one previously, you are good to go!
+# How to Contribute
 
-## Things to Keep in Mind
+To contribute to this project, please follow these steps:
 
-This project uses a **commit then review** process, which means that for approved maintainers, changes can be merged immediately, but will be reviewed by others.
+1. **Run the backup and reset e2e workflow**
+   - Navigate to the Actions tab in the GitHub repository
+   - Find and run the "Backup and Reset to E2E Workflow"
+   - This will create a backup branch and reset the main branch to the E2E commit
 
-For other contributors, a maintainer of the project has to approve the pull request.
+2. **Pull the latest changes to your branch**
+   ```bash
+   git pull origin main
+   ```
 
-# Before You Contribute
+3. **Create a Pull Request**
+   - Make your changes on a feature branch
+   - Push your changes to the repository
+   - Create a Pull Request targeting the main branch
 
-* Check that there is an existing issue in GitHub issues
-* Check if there are other pull requests that might overlap or conflict with your intended contribution
+4. **Update the E2E workflow commit hash**
+   - Once your PR is merged into main, copy the commit hash of the merge commit
+   - Navigate to the repository Settings → Secrets and variables → Actions
+   - Update the repository secret `E2E_WORKFLOW_COMMIT` with the new commit hash
```

Comment:
Curious why it was a secret.  If it was a simple property, people could verify what commit is being saved there.